<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RenameModel</class>
 <widget class="QDialog" name="RenameModel">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>630</width>
    <height>290</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>630</width>
    <height>290</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>630</width>
    <height>290</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Rename Model</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normalon>:/Resources/rename.png</normalon>
   </iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="topPanel" native="true">
     <property name="styleSheet">
      <string notr="true">#topPanel { background-color: qlineargradient(spread:reflect, x1:0.5, y1:0, x2:0, y2:0, stop:0 rgba(91, 204, 233, 100), stop:1 rgba(32, 80, 96, 100)); }</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="modelName">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>25</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font-weight: bold;
padding: 5px 5px 5px 5px;</string>
        </property>
        <property name="text">
         <string>Model Name</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="modelNameDescription">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>25</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">padding: 0px 0px 0px 15px;</string>
        </property>
        <property name="text">
         <string>Define a new name for your model.</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_4">
     <property name="leftMargin">
      <number>12</number>
     </property>
     <property name="topMargin">
      <number>12</number>
     </property>
     <property name="rightMargin">
      <number>12</number>
     </property>
     <item row="1" column="1">
      <widget class="QLabel" name="error_message">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">color: red;</string>
       </property>
       <property name="text">
        <string>Error</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="name">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>Name:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="namelineEdit">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>25</height>
        </size>
       </property>
       <property name="maxLength">
        <number>40</number>
       </property>
       <property name="placeholderText">
        <string>e.g. mymodel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QWidget" name="footerPanel" native="true">
     <property name="styleSheet">
      <string notr="true">#footerPanel {
  background-color: rgb(227, 227, 227);
}</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="cancelButton">
        <property name="focusPolicy">
         <enum>Qt::TabFocus</enum>
        </property>
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="okButton">
        <property name="text">
         <string>OK</string>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
        <property name="default">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>cancelButton</tabstop>
  <tabstop>okButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
